pkgs/by-name/op/openapi-python-client/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenv,
+  darwin,
+  python3Packages,
+  fetchFromGitHub,
+  installShellFiles,
+  ruff,
+  testers,
+  openapi-python-client,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "openapi-python-client";
+  version = "0.21.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    inherit version;
+    owner = "openapi-generators";
+    repo = "openapi-python-client";
+    rev = "v${version}";
+    hash = "sha256-aYwwj96sHfYf3u5Rpcq7QPZA7O4Kt9vVglV38glxuZ0=";
+
+  };
+
+  # ruff is not packaged as a python module in nixpkgs
+  pythonRemoveDeps = [ "ruff" ];
+
+  dependencies =
+    (with python3Packages; [
+      attrs
+      httpx
+      jinja2
+      pydantic
+      python-dateutil
+      ruamel-yaml
+      shellingham
+      typer
+      typing-extensions
+    ])
+    ++ [ ruff ];
+
+  build-system = with python3Packages; [
+    hatchling
+  ];
```

Comment:
According to me, the order should be:
- `nativeBuildInputs`
- `build-system`
- `dependencies`
- `pythonRemoveDeps`

I might not be 100% right though.

Reply:
cool, this makes sense, changed this and updated to 0.21.5